src/main/java/com/keeper/homepage/domain/member/application/MemberService.java

```diff
@@ -60,4 +62,9 @@ public void unfollow(Member member, long otherId) {
     Member other = memberFindService.findById(otherId);
     member.unfollow(other);
   }
+
+  public void updateProfile(Member member, Profile newProfile) {
```

Comment:
https://github.com/KEEPER31337/Homepage-Back-R2/pull/310#discussion_r1299295740

```suggestion

  @Transactional
  public void updateProfile(Member member, Profile newProfile) {
```

Reply:
@Transactional은 DB에 변동사항이 생길 때 붙는 걸로 아는데 해당 기능이 '프로필을 업데이트 하는 기능'인 걸 개발자가 알기 때문에 붙이는 거죠?😮

Reply:
개발자가 안다는 표현이 이해가 안되는데 조금 더 설명해주실 수 있을까요? 

> @transactional은 DB에 변동사항이 생길 때 붙는 걸로 아는데

맞습니다. INSERT나 UPDATE가 일어나면서 한 사이클로 봐야하면 붙여줘야합니당.